app-backup/b2/b2-2.1.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5..8} )
```

Comment:
5? Get outta town 😃 

Do 7,8,9 if you can.

Reply:
I copied the supported versions that were listed on pypi.

Is it preferable to use a gentoo-specific list? The max version listed was 3.8, wants the criteria for including higher versions in am ebuild?